bigtable/CMakeLists.txt

```diff
@@ -35,6 +35,46 @@ if(NOT MSVC)
   endif()
 endif()
 
+# For the gcc and clang compiler families enable a Coverage build type.
```

Comment:
For the GCC and Clang compiler families, enable a Coverage build type.

Reply:
Done.